plugins/modules/events_and_notifications_workflow_manager.py

```diff
@@ -4868,6 +5032,15 @@ def get_diff_deleted(self, config):
             syslog_notification_details = self.want.get('syslog_event_notification')
             syslog_notification_name = syslog_notification_details.get('name')
 
+            if not syslog_notification_name:
+                self.status = "failed"
+                self.msg = (
```

Comment:
self.msg = "A name is a required parameter for deleting syslog events subscription notification in Cisco Catalyst Center."

Need a space b/w notification and in

Reply:
updated.